modules/sharedaddy/sharing-sources.php

```diff
@@ -266,11 +266,15 @@ public function get_amp_display( $post ) {
 	 * @param array $attrs Custom attributes for rendering the social icon.
 	 */
 	protected function build_amp_markup( $attrs = array() ) {
+		$sharer	 = new Sharing_Service();
+		$global	 = $sharer->get_global_options();
 		$attrs        = array_merge(
 			array(
 				'type'   => $this->get_id(),
 				'height' => '32px',
 				'width'  => '32px',
+				'aria-label' => esc_html( $global['sharing_label'] ) .' '. ucfirst($this->shortname),
+				'title' => esc_html( $global['sharing_label'] ) .' '. ucfirst( $this->shortname),
```

Comment:
No need for `esc_html()` here as `esc_attr()` is used below:

```suggestion
				'aria-label' => $global['sharing_label'] ) .' '. ucfirst($this->shortname),
				'title' => $global['sharing_label'] ) .' '. ucfirst( $this->shortname),
```

I'm also unsure about the use of `shortname` here. There should probably be some new property for the label that has a translated string.

Reply:
Great idea about add a new property. Thank you for your suggestion!

Reply:
In fact we have all the translated strings we need at `get_display`. I'll change the code to use it at `get_amp_display` too:

![image](https://user-images.githubusercontent.com/4308648/89462111-f4998c00-d73a-11ea-959f-9aebc3786104.png)